src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -0,0 +1,89 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel;
+
+import edu.wpi.first.networktables.GenericSubscriber;
+import edu.wpi.first.wpilibj.shuffleboard.BuiltInWidgets;
+import edu.wpi.first.wpilibj.smartdashboard.SendableChooser;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import frc.robot.commands.TeleopArmCommand;
+import frc.robot.input.MoInput;
+import frc.robot.utils.MoPrefs;
+import frc.robot.utils.MoShuffleboard;
+import frc.robot.utils.MoSparkMaxPID;
+import frc.robot.utils.TunerUtils;
+import frc.robot.utils.MoSparkMaxPID.Type;
+
+public class ArmSubsystem extends SubsystemBase {
+    public final SendableChooser<Command> armChooser = new SendableChooser<>();
+
+    private final CANSparkMax leftShoulder = new CANSparkMax(
+        Constants.ARM_SHOULDER_LEFT.address, CANSparkMaxLowLevel.MotorType.kBrushless);
+    private final CANSparkMax rightShoulder = new CANSparkMax(
+        Constants.ARM_SHOULDER_RIGHT.address, CANSparkMaxLowLevel.MotorType.kBrushless);
+    private final CANSparkMax wrist = new CANSparkMax(
+        Constants.ARM_WRIST.address, CANSparkMaxLowLevel.MotorType.kBrushless);
+
+    private final MoSparkMaxPID shoulderVelocityPID = new MoSparkMaxPID(Type.VELOCITY, leftShoulder);
+    private final MoSparkMaxPID wristVelocityPID = new MoSparkMaxPID(Type.VELOCITY, wrist);
+
+    private final TeleopArmCommand directArmControlCommand;
+    private final TeleopArmCommand pidArmControlCommand;
+
+    public ArmSubsystem(MoInput input) {
+        rightShoulder.setInverted(true); // TODO: Verify
+        rightShoulder.follow(leftShoulder);
```

Comment:
One nitpick here: there's an overload of `.follow()` that allows you to specify that it should follow inverted. Might be a bit cleaner than two lines, but (assuming `.follow()` respects a previous call to `.setInverted()`) it's not a big deal.